sys/include/crypto/helper.h

```diff
@@ -50,12 +50,18 @@ void crypto_block_inc_ctr(uint8_t block[16], int L);
 int crypto_equals(uint8_t *a, uint8_t *b, size_t len);
 
 /**
- * @brief   Secure wipe function
+ * @brief   Secure wipe function.
+ *
+ * This wipe function zeros the supplied buffer in a way that the compiler is
+ * not allowed to optimize. This can be used to erase secrets from memory.
+ *
+ * Note that this function on its owm could be insufficient. It is outside the
```

Comment:
maybe add:

`.. could be insufficient against (data remanence) attacks.`

This clarifies why it is insufficient.

Reply:
Fixed, thanks for the suggestion